weighter.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
+import os
+import ROOT as r
+import shlex, subprocess
+import argparse
+
+EOS_PATH = "root://eoslhcb.cern.ch/eos/ship/data/IS_experiment"
```

Comment:
`eoslhcb.cern.ch` is not used anymore by SHiP. We have moved to `eospublic.cern.ch//eos/experiment/ship/...`